README.md

```diff
@@ -23,7 +23,7 @@ Currently this repo works with the following Ruby version:
 ```bash
 ➞  ruby -v
```

Comment:
It would also be reasonable to just remove this section from the readme. The convention is to infer the ruby version from the `.ruby-version` file. Thoughts? Maybe it's worth leaving in here since our team doesn't have much ruby experience?

Reply:
oh hm, can prob remove, maybe just replace with a comment that .ruby-version file is where you find the version, but i'm sure folks can probably figure out ruby version via google either way, so totally fine by me either way!

Reply:
I'm more than happy to remove it. Just one less thing to remember to update or that goes out of sync. (Seeing as I just forgot to update it when I bumped the ruby version)